src/libs/Permissions.ts

```diff
@@ -58,6 +58,10 @@ function canUseTableReportView(betas: OnyxEntry<Beta[]>): boolean {
     return !!betas?.includes(CONST.BETAS.TABLE_REPORT_VIEW) || canUseAllBetas(betas);
 }
 
+function canUseHelpSidePanel(betas: OnyxEntry<Beta[]>): boolean {
+    return !!betas?.includes(CONST.BETAS.HELP_SIDE_PANEL) || canUseAllBetas(betas);
+}
```

Comment:
So this isn't used anywhere? If so let's remove it.

Reply:
We have a tracking issue to remove the beta when it's no longer needed anyway, not blocking on this.

Reply:
Ahh, I missed that one 🤦 I should've used this function as the selector of useOnyx. Will do it for Stage 4